static/coffee/flows/services.coffee

```diff
@@ -459,7 +468,7 @@ app.factory 'Flow', ['$rootScope', '$window', '$http', '$timeout', '$interval',
         # { type: 'pause', verbose_name: 'Pause the flow', ivr:true, text:true},
       ]
```

Comment:
Feels weird to have Split by message form be an option here. This is a feature for parsing delimited SMS forms which is also a "wait" ruleset. The only waits here should be USSD based.